bentoml/deployment/serverless/aws_lambda.py

```diff
@@ -106,6 +108,7 @@ def generate_serverless_configuration_for_aws_lambda(
             "useDownloadCache": True,
             "useStaticCache": True,
             "dockerizePip": True,
+            "layer": True,
```

Comment:
is this ok to turn on? I thought we need more customization on the lambda layer

Reply:
I will remove it along with the validation check for api name